README.md

```diff
@@ -1,4 +1,23 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four other blocks, with which it is lying side to side.
```

Comment:
"with which it is lying side to side."
This reads weird. Can we rephrase it?
"up to four other neighbouring blocks"